src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -783,6 +805,8 @@ event::ptr primitive_inst::execute(const std::vector<event::ptr>& events) {
             return ev;
```

Comment:
So here, 
1) if output layout is empty 
2) set prim_inst->can_be_optimized = true 
3) run on_execute()
4) set prim_inst->can_be_optimized = false

Reply:
Updated. Thanks!